bitcoin-test/package.yaml

```diff
@@ -0,0 +1,47 @@
+name: bitcoin-test
+version: 0.1.0
+synopsis: Bitcoin test library for Haskell
+description: Please see the README on GitHub at <https://github.com/haskell-bitcoin/bitcoin#readme>
+category: Bitcoin, Finance, Network
+author:
+  - Philippe Laprade
+  - Jean-Pierre Rupp
+  - Matthew Wraith
+  - Keagan McClelland
+maintainer: keagan.mcclelland@gmail.com
+license: BSD3
+homepage: http://github.com/haskell-bitcoin/bitcoin#readme
+git: git://github.com/haskell-bitcoin/bitcoin.git
+bug-reports: http://github.com/haskell-bitcoin/bitcoin/issues
+extra-source-files:
+  - ./data/*.json # raw data for tests
+dependencies:
+  - bitcoin
```

Comment:
This should prob have a version, same for the bench package if applicable.

Reply:
Why, shouldn't it always be in sync with the one in the repo?

Reply:
It depends on if we want to support users directly depending on the test package.  I have found that it generally helps to have a lot of helper functions and maybe even a framework for testing bitcoin.  We could treat `bitcoin-test` as a product, which exports that kind of stuff.  It would be too opinionated to export directly from `bitcoin`, but that's ok.  However, if we just want this package to be the unit tests, then I agree a version is not necessary.

After writing this note, I do think that `bitcoin-test` should just be for unit tests, so no version necessary.

Reply:
so let's send it?